app/controllers/hosts_controller.rb

```diff
@@ -512,15 +513,29 @@ def submit_rebuild_config
     end
 
     if message.blank?
-      success _('Configuration successfully rebuilt')
+      respond_to do |format|
+        format.html do
+          process_success(:success_redirect => helpers.current_hosts_path, :success_msg => _('Configuration successfully rebuilt'))
+        end
+        format.json do
+          render :json => { :success_msg => _('Configuration successfully rebuilt') }
+        end
+      end
+      success
     else
-      error message
+      respond_to do |format|
+        format.html do
+          process_error(:redirect => helpers.current_hosts_path, :error_msg => message)
+        end
+        format.json do
```

Comment:
Is it me, or is the indenting off?

Reply:
I didnt see a rubocop error ?

Reply:
Now it looks ok. I think the GH app sometimes doesn't render it well